index.js

```diff
@@ -1,18 +1,16 @@
 import https from "https";
 import fs from "fs";
-import dotenv from "dotenv";
-
-dotenv.config();
-
-const { PORT } = process.env;
+import config from "config";
 
 import { app } from "./app.js";
 
+const port = config.get("server.port");
+
 const key = fs.readFileSync("./key.pem");
 const cert = fs.readFileSync("./cert.pem");
 
 const server = https.createServer({ key: key, cert: cert }, app);
 
-server.listen(PORT || 3000, () => {
-  console.log("Server is running");
+server.listen(port || 4000, () => {
```

Comment:
redundant default, config.get("server.port") will throw error if port is missing